packages/viewer/src/components/viewer-markup-arrow/viewer-markup-arrow.css

```diff
@@ -0,0 +1,170 @@
+   */
+  --viewer-markup-arrow-head-stroke-width: 4;
+  /**
+   * @prop --viewer-markup-arrow-head-fill-color: A CSS color that
+   * specifies the color of the arrow head's fill.
+   */
+  --viewer-markup-arrow-head-fill-color: var(--red-800);
+  /**
+   * @prop --viewer-markup-arrow-head-fill-opacity: A number between 
+   * 0 and 1 that specifies the opacity of the arrow head's fill.
+   */
+  --viewer-markup-arrow-head-fill-opacity: 1;
+  /**
+   * @prop --viewer-markup-arrow-bounds-cap-border-color: A CSS color that
+   * specifies the color of arrow start/end resize anchors' border.
+   */
+  --viewer-markup-arrow-bounds-cap-border-color: var(--blue-400);
+  /**
+   * @prop --viewer-markup-arrow-bounds-cap-border-width: A CSS length that
+   * specifies the width of arrow start/end resize anchors' border.
+   */
+  --viewer-markup-arrow-bounds-cap-border-width: 1px;
+  /**
+   * @prop --viewer-markup-arrow-bounds-cap-background-color: A CSS color that
+   * specifies the background color of the arrow start/end resize anchors.
+   */
+  --viewer-markup-arrow-bounds-cap-background-color: white;
+  /**
+   * @prop --viewer-markup-arrow-bounds-center-anchor-border-color: A CSS color that
+   * specifies the color of the center resize anchor's border.
+   */
+  --viewer-markup-arrow-bounds-center-anchor-border-color: white;
+  /**
+   * @prop --viewer-markup-arrow-bounds-center-anchor-border-width: A CSS length that
+   * specifies the width of the center resize anchor's border.
+   */
+  --viewer-markup-arrow-bounds-center-anchor-border-width: 1px;
+  /**
+   * @prop --viewer-markup-arrow-bounds-center-anchor-background-color: A CSS color that
+   * specifies the background color of the center resize anchor.
+   */
+  --viewer-markup-arrow-bounds-center-anchor-background-color: var(--blue-400);
+  /**
+   * @prop --viewer-markup-arrow-bounds-outline-width: A CSS color that
+   * specifies the width of the selected arrow outline.
+   */
+  --viewer-markup-arrow-bounds-outline-width: 1px;
+  /**
+   * @prop --viewer-markup-arrow-bounds-outline-color: A CSS color that
+   * specifies the color of selected arrow outline.
+   */
+  --viewer-markup-arrow-bounds-outline-color: var(--blue-400);
+  /**
+   * @prop --viewer-markup-arrow-bounds-anchor-width: A CSS length that
+   * specifies the width of the resize and reposition anchors.
+   */
+   --viewer-markup-arrow-bounds-anchor-width: 9px;
+   /**
+    * @prop --viewer-markup-arrow-bounds-anchor-height: A CSS length that
+    * specifies the height of the resize and reposition anchors.
+    */
+   --viewer-markup-arrow-bounds-anchor-height: 9px;
+  ;
+}
+
+.svg {
+  width: 100%;
+  height: 100%;
+}
+
+.line {
+  pointer-events: auto;
+  stroke: var(--viewer-markup-arrow-line-stroke-color);
+  stroke-width: var(--viewer-markup-arrow-line-stroke-width);
+  fill: var(--viewer-markup-arrow-line-fill-color);
+  fill-opacity: var(--viewer-markup-arrow-line-fill-opacity);
```

Comment:
Is there ever a case of filling a line?

Reply:
Yeah, I suppose this probably isn't needed. We set these in mfg-ui, but I don't know of a case where the `line` element could display that fill. I'll remove this